en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md

```diff
@@ -212,9 +212,10 @@ Let's deploy a front-end application to consume the API. This application is des
 Once you create the web application component, you can deploy it to the Choreo runtime. To deploy the web application component, follow the steps below:
 
 1. In the left menu, click **Deploy**.
-2. In the **Build Area** card, click **Build and Deploy**. The deployment may take a few minutes to complete.
-3. Once you deploy the web application, copy the **Web App URL** from the development environment card.
-4. Navigate to the web app URL. You can verify that you have successfully hosted the web application.
+2. In the **Build Area** card, click **Configure & Deploy**. 
+3. Add configurations and deploy. The deployment may take a few minutes to complete. See how to define and read configurations at [Develop a Web Application page](https://wso2.com/choreo/docs/develop-components/develop-a-web-application/#creating-a-web-application).
```

Comment:
What configurations are these? And what values do users have to add?

Reply:
Please refer this comment to view document changes request - https://github.com/wso2-enterprise/choreo/issues/23014#issuecomment-1689624405